docs/user/device-groups.md

```diff
@@ -0,0 +1,43 @@
+---
+navTitle: Device Groups
+---
+
+# Device Groups
+
+
+**Navigation**: `Team > Application > Device Groups`
+
+Device groups allow you to organise your Application assigned devices into logical groups.
+These groups can be the target of [DevOps Pipelines](./devops-pipelines.md), greatly
```

Comment:
Is there another way to deploy to these groups? `can` is implying that for me. In other words, is there tight coupling with devops pipelines or not?